backend/index.ts

```
@@ -1 +1,24 @@
+import * as dotenv from 'dotenv';
+import { weaviateRoute } from './weaviateRoute'; 
+
+dotenv.config;
+
+const weaviate_key = process.env.WEAVIATE_API_KEY ?? '';
+const cohere_key = process.env.COHERE_API_KEY ?? '';
+
+if (!weaviate_key) {
+    console.log("API KEY WAS NOT PROVIDED");
```

throw error instead of console log